nix/sources.json

```diff
@@ -25,12 +25,12 @@
         "builtin": false,
         "description": "Internet Computer blockchain source: the client/replica software run by nodes",
         "homepage": "",
-        "owner": "dfinity",
+        "owner": "ggreif",
```

Comment:
this is just an experiment for @crusso, hopefully unblocking him. This will be reverted before merge. Please ignore.